### PR TITLE
Translate badge to english

### PR DESCRIPTION
**Pull request by afc163**
#1471

### CODE REVIEW

**Comment by afc163** [comment]
@waywardmonkeys @manV

**Comment by manV** [comment]
Looks good to me.

File: components/badge/demo/change.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+  en-US: Dynamic
 ---
 
+## zh-CN
+
 展示动态变化的效果。
 
+## en-US
+
+Count will change with animation.
```
Comment (waywardmonkeys) [inline components/badge/demo/change.md]: The count will be animated as it changes.

File: components/badge/demo/dot.md [hunk 2]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 3
-title: 讨嫌的小红点
+title:
+  zh-CN: 讨嫌的小红点
+  en-US: Red dot
```
Comment (waywardmonkeys) [inline components/badge/demo/dot.md]: Red badge

File: components/badge/demo/dot.md [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
 ---
 
+## zh-CN
+
 没有具体的数字。
 
+## en-US
+
+Just Red dot without specific count.
```
Comment (waywardmonkeys) [inline components/badge/demo/dot.md]: This will simply display a red badge, without a specific count.

File: components/badge/demo/link.md [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
+  en-US: Clickable
 ---
 
+## zh-CN
+
 用 a 标签进行包裹即可。
 
+## en-US
+
+Badge could be wrapped by tag `a`.
```
Comment (waywardmonkeys) [inline components/badge/demo/link.md]: The badge can be wrapped with an `a` tag to make it clickable.

File: components/badge/index.en-US.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+```jsx
+<Badge count={5} />
+```
+
+| Property       | Description             | Type       | Default |
+|----------------|-------------------------|------------|---------|
+| count          | Number to show in badge | Number     |         |
+| overflowCount  | Max count to show       | Number     | 99      |
+| dot            | whether to show red dot without number | Boolean | false  |
```
Comment (waywardmonkeys) [inline components/badge/index.en-US.md]: Hmm. I guess we're stuck with that usage of 'dot' for here / now.

Comment (afc163) [inline components/badge/index.en-US.md]: Any sugguestion? I think `dot` has a clear meaning for its usage.

**Comment by afc163** [comment]
Updated and Thx! @waywardmonkeys

**Pull request merged**